nixos/modules/services/misc/home-assistant.nix

```diff
@@ -251,16 +251,54 @@ in {
         User = "hass";
         Group = "hass";
         Restart = "on-failure";
+        KillSignal = "SIGINT";
+
+        # Hardening
+        AmbientCapabilities = lib.mkForce [
+          # required for bluetooth device interaction (e.g. bluetooth tracker component)
+          "cap_net_raw"
+          "cap_net_admin"
+        ];
+        CapabilityBoundingSet = [
+          "cap_net_raw"
+          "cap_net_admin"
+        ];
+        LockPersonality = true;
+        MemoryDenyWriteExecute = true;
```

Comment:
Do we have any node.js plugins around? This would crash them.

Reply:
I don't think we have.